### PR TITLE
Fix for issue #7

### PR DESCRIPTION
**Pull request by jamonholmgren**
Fixes issue #7, uninitialized constant.

### CODE REVIEW

File: lib/ProMotion-iap.rb [hunk 1]
```diff
@@ -6,7 +6,9 @@
 
 Motion::Project::App.setup do |app|
   lib_dir_path = File.dirname(File.expand_path(__FILE__))
-  app.files << File.join(lib_dir_path, "ProMotion/iap.rb")
-  app.files << File.join(lib_dir_path, "ProMotion/product.rb")
+  app.files.unshift [
+    File.join(lib_dir_path, "ProMotion/iap.rb"),
+    File.join(lib_dir_path, "ProMotion/product.rb"),
+  ]
```
Comment (jamonholmgren) [inline lib/ProMotion-iap.rb]: Adds these files to the beginning, not the end, of the files array.

File: Gemfile [hunk 2]
```diff
@@ -1,5 +1,4 @@
 source 'https://rubygems.org'
 
 # Define all dependencies in your .gemspec file
-gem "ProMotion", github: "clearsightstudio/ProMotion", branch: "master"
```
Comment (jamonholmgren) [inline Gemfile:4]: Remove ProMotion dependency, not needed.

File: lib/ProMotion/product.rb [hunk 3]
```diff
@@ -24,4 +24,4 @@ def restore(&callback)
       end
     end
   end
-end
\ No newline at end of file
+end
```
Comment (kevinvangelder) [inline lib/ProMotion/product.rb]: @jamonholmgren What happened to your `ensure_file_newline_on_save`?

**Comment by kevinvangelder** [comment]
:+1:

**Pull request merged**